src/task-5.js

```diff
@@ -1,16 +1,90 @@
 function getExpeditionsTargets(commandSeries) {
```

Comment:
Do not forget to apple changes here. And I don't mind if you want to import a solution from 'task-4.js'